docs/user/plagiarism-check.rst

```diff
@@ -172,6 +173,73 @@ The process is visualized in the following diagram.
 2. Reply to the message of the instructor.
 3. Wait for the final verdict.
 
+Continuous Plagiarism Control
+-----------------------------------------
+
+In this section, we explain the automated flow of the continuous plagiarism control (CPC) and presents actions taken by from different Artemis users.
+
+Automated Flow
+^^^^^^^^^^^^^^
+
+The CPC is an experimental feature designed to shorten the feedback loop for students and reduce the manual work required from tutors and instructors.
+Continuous feedback on potential plagiarism aims to encourage students to submit their original work and enhance their learning outcomes.
+The continuous plagiarism control **does not accuse students of plagiarism, nor does it affect their scores**.
+Instead, it **notifies students of significant similarities** detected between their submissions, allowing them to revise their solutions.
+
+The CPC initiates plagiarism checks every night for all onboarded exercises, processing all latest submissions for each exercise.
+If a submission does not include significant similarity, the CPC removes any associated existing plagiarism case.
+If significant similarity is detected, the CPC creates or updates a plagiarism case for that submission.
+Subsequently, the student analyzes the feedback received and, if the submission period is still open, can submit a new solution.
+If the submission period has ended, the student can either dispute the plagiarism case or choose to take no action.
+
+The following activity diagram depicts the CPC flow described above.
+
+|cpc-activity-diagram|
+
+Instructors
+^^^^^^^^^^^
+
+Configuration of the CPC
+""""""""""""""""""""""""
+
+Instructors can configure various aspects of the CPC for each exercise. The following list describes all configuration options:
+
+* **enabling the CPC** - Instructor can turn the CPC o n and off for a particular exercise. Default: `OFF`.
+* **enabling the post due date checks** - Instructor can configure the CPC to test the exercise one more time the night after the exercise due date. This can help to test even the just in time submission. Default: `OFF`.
+* **setting the response period for the significant similarity plagiarism case** - Instructor can specify for how long after the exercise due date a student will be able to submit a complaint. Default: `7 days`. Min: `7 days`.
+* **setting the similarity threshold** - Instructor can specify the similarity threshold for the plagiarisms check performed by the CPC. Default: `90%`.
+* **setting the minimum score** - Instructor can specify the minimum submission score for the plagiarisms check performed by the CPC. Default: `0`.
+* **setting the minimum size** - Instructor can specify the minimum submission size for the plagiarisms check performed by the CPC. Default: `50`.
```

Comment:
```suggestion
* **setting the similarity threshold** - Instructor can specify the similarity threshold for the plagiarism check performed by the CPC. Default: `90%`.
* **setting the minimum score** - Instructor can specify the minimum submission score for the plagiarism check performed by the CPC. Default: `0`.
* **setting the minimum size** - Instructor can specify the minimum submission size for the plagiarism check performed by the CPC. Default: `50`.
```

Reply:
How about rewording some of those sentences instead of essentially repeating the bold setting name in a full sentence: E.g.,

```
* **setting the minimum score** - Only submissions that have a minimum score of at least this value are included in the check for other similar submissions. Default: `0`.
* **setting the minimum size** - Submissions smaller than this value will not be checked for similarity to other submissions. Default: `50`.
```

Reply:
Those three options are described above in the manual checks section. Here I want to explicitly mention their name, so that it is clear that these are the same